Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/manager_app/view/child_list/ChildListActivity.java

```diff
@@ -44,11 +51,34 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuInflater menuInflater = getMenuInflater();
+        menuInflater.inflate(R.menu.plan_list_menu, menu);
+        MenuItem searchViewItem = menu.findItem(R.id.menu_search);
```

Comment:
Have you tried you use data binding here? (Similar to save data button)

Reply:
What do you mean?

Reply:
I mean data binding for keeping searchViewItem object without finding it by id or implement setOnQueryTextListener with managing this action by data binding not manually finding it.  Like in PlanCreate but for menu it could be a little different.